index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="/home/asvc/Documents/Meeting2/style.css">

```

Comment:
Please add the proper path to the style.css file

Reply:
- [ ] Kindly always include an explanation, a "why", for each requested change.

Reply:
Please add the proper path to the style.css file to run the files that your style been applied to the html.